packages/fxa-auth-server/lib/oauth/metrics/amplitude.ts

```diff
@@ -30,7 +30,7 @@ const EVENTS = {
 
 const FUZZY_EVENTS = new Map([]);
 
-module.exports = (log: Logger, config: conf) => {
+module.exports = (log: Logger, config: ReturnType<conf['getProperties']>) => {
```

Comment:
We might want to type alias this in `config/index.ts` now that I think about it, to avoid having to do this everywhere.